setup/deb-setup.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+echo 'Setting up your developmental environment. This may take a while.'
+
+echo '[*] Adding keys...'
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+sudo apt-get update
+
+echo '[*] Installing GNUpg...'
+sudo apt-get install gnupg
+
+echo '[*] Adding Keys for rvm...'
+gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB && echo '[*] Installing rvm...' && \curl -sSL https://get.rvm.io | bash -s stable
+
+echo '[*] Installing Ruby-2.5.0...'
+sudo rvm install ruby-2.5.0
```

Comment:
Ruby 2.5.0 will not yet be active on a fresh install. When RVM installer exits, it provides a necessary step in its output, along the lines of `source /home/sage/.rvm/scripts/rvm`.

Only once you do that (or log out and log in again) will rvm be active, and then you can install the required ruby version.